public/locales/en/translation.json

```diff
@@ -141,6 +141,12 @@
           "title": "No scenario yet",
           "label": "You can create a scenario by clicking on \"Create new scenario\""
         },
+        "nodashboard": {
+          "label": "There isn't any dashboard configured for this run template"
+        },
+        "unknownStatus": {
+          "label": "This scenario has the \"Unknown\" status, if the problem persists, please, contact your administrator"
```

Comment:
Same remark as in the "@cosmotech/ui" repository, we could display a simpler message hiding the technical details =)
(in both translation files and in the file _src/views/Scenario/labels.js_)